server/src/main/java/io/druid/segment/realtime/plumber/RealtimePlumber.java

```diff
@@ -363,7 +363,9 @@ private Sink getSink(long timestamp)
                                           }
                                         }
                                     )
-                                )
+                                ),
+                                "query/wait/time",
+                                ImmutableMap.of("segment", theSink.getSegment().getIdentifier())
```

Comment:
Apparently it still does, I really thought it had been removed at one point.

Weather it is or isn't the best thing here is to have similar behavior to the ServerManager workflow. So since ServerManager currently has it this should be fine then.

Looking at the workflow though, shouldn't `query/segmentAndCache/time` be the appropriate name here?